opm/parser/eclipse/EclipseState/IOConfig/RestartConfig.cpp

```diff
@@ -50,10 +51,97 @@ namespace Opm {
                 && std::all_of( x.begin() + 1, x.end(), is_digit );
         }
 
+
+        std::set<std::string> notSupported = {"NORST", "SFREQ" , "STREAM"};
+
+        constexpr const char*  integerKeywords[] = {""     ,      //  1 BASIC=?? - ignored
+                                                    "FLOWS",      //  2
+                                                    "FIP",        //  3
+                                                    "POT",        //  4
+                                                    "PBPD",       //  5
+                                                    "",           //  6 FREQ=?? - ignored
+                                                    "PRES",       //  7
+                                                    "VISC",       //  8
+                                                    "DEN",        //  9
+                                                    "DRAIN",      // 10
+                                                    "KRO",        // 11
+                                                    "KRW",        // 12
+                                                    "KRG",        // 13
+                                                    "PORO",       // 14
+                                                    "NOGRAD",     // 15
+                                                    "NORST",      // 16 NORST - not supported
+                                                    "SAVE",       // 17
+                                                    "SFREQ",      // 18 SFREQ=?? - not supported
+                                                    "ALLPROPS",   // 19
+                                                    "ROCKC",      // 20
+                                                    "SGTRAP",     // 21
+                                                    "",           // 22 - Blank - ignored.
+                                                    "RSSAT",      // 23
+                                                    "RVSAT",      // 24
+                                                    "GIMULT",     // 25
+                                                    "SURFBLK",    // 26
+                                                    "PCOW,PCOG",  // 27 - Multiple, split on "," [Special cased]
+                                                    "STREAM",     // 28 STREAM=?? - not supported
+                                                    "RK",         // 29
+                                                    "VELOCITY",   // 30
+                                                    "COMPRESS" }; // 31
+
+
+        static void insertKeyword(std::set<std::string>& keywords, const std::string& keyword) {
+            const auto pos = keyword.find( "=" );
```

Comment:
There's actually a `basic_string::find( CharT, size_type pos = 0 )` overload. Since we're looking for a character we can use that rather than string-based. Minor, but should be slightly faster.